WordPress/Classes/ViewRelated/Gutenberg/GutenbergNetworking.swift

```diff
@@ -0,0 +1,62 @@
+import Alamofire
+
+struct GutenbergNetworkRequest {
```

Comment:
This is just a bit of a rant, but specially after doing JS for a while, I find interesting how reluctant we all are to use top level functions on Swift. There's no point in rewriting this, but it's something that keeps catching my attention 😁

Reply:
I like the Struct to pack some properties inside it, and have methods with less arguments.
At first I did static functions (that would work as well as top level functions), but I didn't like having all the internal helper functions needing the whole set of arguments.